assets/js/components/product-category-control/index.js

```diff
@@ -158,10 +159,15 @@ ProductCategoryControl.propTypes = {
 	 * The list of currently selected category IDs.
 	 */
 	selected: PropTypes.array.isRequired,
+	/**
+	 * Allow multiple selection. Defaults to true.
+	 */
+	multiple: PropTypes.bool,
```

Comment:
What would you think about renaming this prop from `multiple` to `isSingle` to match how it's named in `SearchListControl`?